docs/articles/expensify-classic/bank-accounts-and-credit-cards/deposit-accounts/Deposit-Accounts-USD.md

```diff
@@ -1,5 +1,74 @@
 ---
 title: Deposit Accounts - USD
-description: Deposit Accounts - USD
+description: How to add a deposit account to receive payments for yourself or your business (US)
 ---
-## Resource Coming Soon!
+# Overview  
+
+There are two types of deposit-only accounts:  
+
+1. If you're an employee seeking reimbursement for expenses you’ve incurred, you’ll add a **Personal deposit-only bank account**.
+2. If you're a vendor seeking payment for goods or services, you’ll add a **Business deposit-only account**.  
+
+# How to connect a personal deposit-only bank account
+
+**Connect a personal deposit-only bank account if you are:**  
+
+- An employee based in the US who gets reimbursed by their employer 
+- An employee based in Australia who gets reimbursed by their company via batch payments
+- An international (non-US) employee whose US-based employers send international reimbursements
+
+**To establish the connection to a personal bank account, follow these steps:**  
+
+1. Navigate to your **Settings > Account > Payments** and click the **Add Deposit-Only Bank Account** button.
+2. Click **Log into your bank** button and click **Continue** on the Plaid connection pop-up window.
+3. Search for your bank account in the list of banks and follow the prompts to sign-in to your bank account.
+4. Enter your bank login credentials when prompted.
+    - If your bank doesn't appear, click the 'x' in the upper right corner of the Plaid pop-up window and click **Connect Manually**.
+    - Enter your account information, then click **Save & Continue**.
+
+You should be all set! You’ll receive reimbursement for your expense reports directly to this bank account. 
+
+# How to connect a business deposit-only bank account  
+
+**Connect a business deposit-only bank account if you are:**  
+
+- A US-based vendor who wants to be paid directly for bills sent to customers/clients
+- A US-based vendor who want to pay invoices directly via Expensify
```

Comment:
Wait, is this right? Wouldn't you need a withdrawal VBA in order to pay invoices directly via Expensify?

Reply:
They can add this as a deposit account and then click "Verify" 😃